deployments/docker/src/load-config.mk

```diff
@@ -11,6 +11,7 @@ VTS_PORT ?= 50051
 PROVISIONING_PORT ?= 8888
 VERIFICATION_PORT ?= 8080
 MANAGEMENT_PORT ?= 8088
+KEYCLOAK_PORT ?= 11111
```

Comment:
Just a thought, this could be 8880 ?

Reply:
I specifically chose something different, since this isn't really part of Veraison -- this is an external service Veraison depends on (besides, its already difficult enough to remember the correct port to use for DEBUG_PORT, as they're all just 8's and 0's in various permutations).